locales/en-US/editor.properties

```diff
@@ -116,6 +116,7 @@ CMD_FILE_NEW=New File
 CMD_FILE_NEW_FOLDER=New Folder
 CMD_FILE_RENAME=Rename
 CMD_FILE_DELETE=Delete
+CMD_FILE_DOWNLOAD=Download
```

Comment:
Ok, I'll do a PR on thimble with the fix mentioned.
Also, I added 2 more strings
`UNEXPECTED_ERROR` and `ERROR_DIALOG_HEADER`